docs/developers/getting_started.rst

```diff
@@ -6,19 +8,33 @@ Checking Out The Codebase
 
 The master code repository is available on Github:  http://github.com/imageworks/OpenColorIO
 
-For those unfamiliar with git, the wonderful part about it is that even though only a limited number people have write access to the master repository, anyone is free to create, and even check in, changes to their own local git repository.  Your local changes will not automatically be pushed back to the master repository, so everyone feel free to informally play around with the codebase.  Also - unlike svn - when you download the git repository you have a full copy of the project's history (including revision history, logs, etc), so the majority of code changes you will make, including commits, do not require network server access.
+For those unfamiliar with git, the wonderful part about it is that even though
+only a limited number people have write access to the master repository, anyone
+is free to create, and even check in, changes to their own local git repository. 
+Your local changes will not automatically be pushed back to the master
+repository, so everyone feel free to informally play around with the codebase.
+Also - unlike svn - when you download the git repository you have a full copy of
+the project's history (including revision history, logs, etc), so the majority
+of code changes you will make, including commits, do not require network server
+access.
 
-The first step is to install git on your system.  For those new to the world of git, github has an excellent tutorial stepping you through the process, available at: http://help.github.com/
+The first step is to install git on your system.  For those new to the world of
+git, github has an excellent tutorial stepping you through the process,
```

Comment:
I would always use the correct name GitHub rather than github in the plain text.